src/components/site-logo/site-logo.module.css

```diff
@@ -0,0 +1,8 @@
+.icon {
+  max-height: 1rem;
```

Comment:
`rem` is like `px`, it's a unit of size. But i prefer `rem` because it scales with the font-size of whatever it's next to. I think.